src/components/animated-button/AnimatedButtonProvider.tsx

```diff
@@ -70,8 +86,8 @@ const AnimatedButton = ({
 
 export type AnimatedButtonProviderProps = {
   isDarkMode: boolean
-  iconDarkJSON: () => any
-  iconLightJSON: () => any
+  iconDarkJSON: any
```

Comment:
probably I did this, but the any was necessary b/c the JSON wasn't well typed. maybe we could at least do

`type JSON = any` or (object?) and reuse that here or something